Resources/Textures/Parallaxes/licences.txt

```diff
@@ -3,4 +3,6 @@ Licensed under MIT at https://github.com/cpetry/TextureGenerator-Online/blob/gh-
 
 noise.png generated from https://giggster.com/guide/cloud-texture-generator/
 
-KettleParallaxNeb.png and KettleParallaxBG.png adapted from Screaming Brain Studios CC-0 https://opengameart.org/content/seamless-space-backgrounds
\ No newline at end of file
+KettleParallaxNeb.png and KettleParallaxBG.png adapted from Screaming Brain Studios CC-0 https://opengameart.org/content/seamless-space-backgrounds
+
+planet.png from tgstation at commit https://github.com/tgstation/tgstation/commit/f68af3ee3a34e07ad92125a8ee3b49193c0bc8ea
```

Comment:
These should be using RGA (the other ones predate it).

Reply:
wtf does that mean

Reply:
look for attributions.yml

Reply:
https://github.com/space-wizards/RobustToolbox/blob/06c64b3be8e52d52511624447cffeff551cbb262/Schemas/rga.yml